src/screens/CreatePost/elements/CreatePollContainer/CreatePollContainer.ios.js

```diff
@@ -63,13 +65,18 @@ function CreatePollContainer({
           onpollchanged={(v) => {
             onsinglepollchanged(v, index);
           }}
+          isAnonym={isAnonym}
         />
       ))}
 
       {polls.length < MAX_POLLING_ALLOWED && (
         <TouchableOpacity onPress={() => onaddpoll()} style={S.addpollitemcontainer}>
-          {/* TODO: Garry warna sesuai mode */}
-          <MemoIcPlus width={16} height={16} style={S.addpollitemplusicon} fill={COLORS.white} />
+          <MemoIcPlus
+            width={20}
+            height={20}
+            style={S.addpollitemplusicon}
+            fill={isAnonym ? COLORS.anon_secondary : COLORS.signed_secondary}
+          />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isAnonym` prop is being used to conditionally set the fill color of the `MemoIcPlus` component. This is a good practice as it allows for dynamic styling based on the state of the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->